Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -221,7 +221,7 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['DigiFullPU']=1
             wmsplit['RecoFullPU']=1
             wmsplit['RECOHID11']=1
-            wmsplit['DigiFullPU_2023D17PU']=1
+            wmsplit['DigiFullPU_2023D17PU'] = 1 
```

Comment:
you didn't actually revert properly here - this will cause a conflict with your other PR
I would advise just removing all commits that edit this file from your branch entirely